include/cloudbreak.bash

```diff
@@ -202,11 +202,13 @@ cloudbreak-conf-defaults() {
         env-import CAAS_URL $(service-url auth-mock "$BRIDGE_ADDRESS" "$CB_LOCAL_DEV_LIST" "" "8080" "10080")
         if [[ "$UMS_ENABLED" == "true" ]]; then
             env-import UMS_HOST $(service-url auth-mock "$BRIDGE_ADDRESS" "$CB_LOCAL_DEV_LIST" "" "" "")
+            env-import GATEWAY_DEFAULT_REDIRECT_PATH "/environments"
         fi
     else
         env-import CAAS_URL $(service-url caas-api "$BRIDGE_ADDRESS" "$CB_LOCAL_DEV_LIST" "" "8080" "10080")
         if [[ "$UMS_ENABLED" == "true" ]]; then
             env-import UMS_HOST $(service-url caas-api "$BRIDGE_ADDRESS" "$CB_LOCAL_DEV_LIST" "" "" "")
+            env-import GATEWAY_DEFAULT_REDIRECT_PATH "/cloud"
```

Comment:
what if DPS is enabled but UMS isn't? GATEWAY_DEFAULT_REDIRECT_PATH would be empty

Reply:
Moved it to the outer layer.